src/oneTrust/endpoints/getAssessment.ts

```diff
@@ -185,18 +176,7 @@ export const OneTrustAssessmentNestedQuestion = t.type({
   /** Indicates whether the question is valid. */
   valid: t.boolean,
   /** Type of question in the assessment. */
-  type: t.union([
-    t.literal('TEXTBOX'),
-    t.literal('MULTICHOICE'),
-    t.literal('YESNO'),
-    t.literal('DATE'),
-    t.literal('STATEMENT'),
-    t.literal('INVENTORY'),
-    t.literal('ATTRIBUTE'),
-    t.literal('PERSONAL_DATA'),
-    t.literal('ENGAGEMENT'),
-    t.literal('ASSESS_CONTROL'),
-  ]),
+  type: t.string,
```

Comment:
@abrantesarthur Should this be a union with `null`? `null` was not in the union before.

Reply:
I don't think so! In addition, If we convert this to string, it'll affect [getQuestionTypeFromOneTrustAssessment](https://github.com/transcend-io/main/blob/94e357b84384c8e5656e2f6c93a7f75cbf5e4ed9/packages/backend/src/models/assessmentForm/helpers/importOneTrustAssessmentForms/getQuestionTypeFromOneTrustAssessment.ts#L10). It won't break it, though, so up to you.

Reply:
sounds good!

Reply:
honestly, might as well do a union with null. Again, I don't think you have to because this didn't break until now, but you never know with OT lol.